app/src/containers/Dashboard/TrialBannerAndModal.tsx

```
@@ -339,43 +313,104 @@ const TrialBannerAndModal: React.FC<any> = ({
+    if (allDomains) {
+      if (domainName) {
+        const newdomain = allDomains.getUserSites.filter(
+          (site: any) => site.url == domainName,
+        )[0];
+        setDomainCount(allDomains.getUserSites.length);
+        if (newdomain) {
+          setAddedDomain(newdomain);
+        }
+      }
+      let trialSites = 0;
+      for (let site of allDomains?.getUserSites) {
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)